index.html

```diff
@@ -277,10 +277,25 @@ <h3>Normative Specifications</h3>
                             <b>Expected completion:</b> WG-START + 24 months.
                         </p>
                     </dd>
-                    <dt id="signatures" class="spec">Linked Data Security (LDS)</dt>
+
+                    <dt id="signatures" class="spec">Linked Data Hash (LDH)</dt>
```

Comment:
To start -- I won't object to this; I could live with it.

I do feel like it's unnecessary and the description here could be folded into the Linked Data Integrity specification. While it's true that the steps are 1) canonize, 2) hash, 3) sign/protect -- step 2 is a 1 page specification, if we choose to split it out. I don't think it's necessary to split it out if we choose "Linked Data Integrity".

Reply:
You are probably right that this 'LDH' spec might be very short. However, the preamble of this section does say:

> Note that the titles of the documents are tentative and not final. The Working Group may also decide to either split some of those documents or, conversely, merge some of them. 

So, at the end of the day, we may decide to merge (or further subdivide) some of these planned document. 

For the time being, I see this list of deliverables in the charter at least as much as a way of clearly separating the technical "chapters" of a large, conceptual deliverable, as a list of specific documents that the WG will, eventually, publish. The discussions we were having so far has convinced me that this separation is really necessary to avoid further discussions when the charter ends up being voted upon.

Reply:
The WG will have flexibility here with naming and shuffling things around, so I'm ok with it -- we're just trying to capture the concepts and I think this does.